### ISSUE TITLE
Is mldb a full-featured end-to-end system?

### ISSUE THREAD

**Issue by Make42** [mldbai/mldb#965]
Because of the "database" in the name, I first thought that mldb is something like a feature store. After reading a bit more it seems to me, that - instead - it is a full out-of-the-box, batteries-included type of system, where I am supposed to use the provided ML implementations (even though I can write plug-ins for the system). Which impression is correct?

My current understand is this: I use the mldb-SQL queries to define a DAG (directed acyclic graph) of data processing (including ML), the input data and the output REST ports. Is this right?

**Comment by jeremybarnes** [comment]
Yes, that was the vision for MLDB.  Since it was acquired in 2017 the system has stagnated.  It's the base for research into the architecture of ML, but I would not use it for anything apart from design inspiration now.
